lttoolbox/alphabet.cc

```diff
@@ -249,20 +250,20 @@ Alphabet::getSymbol(wstring &result, int const symbol, bool uppercase) const
 }
 
 bool
-Alphabet::isTag(int const symbol) const
+Alphabet::isTag(int32_t const symbol) const
 {
   return symbol < 0;
 }
 
-pair<int, int> const &
-Alphabet::decode(int const code) const
+pair<int32_t, int32_t> const &
+Alphabet::decode(int32_t const code) const
 {
   return spairinv[code];
 }
 
-set<int>
-Alphabet::symbolsWhereLeftIs(wchar_t l) const {
-  set<int> eps;
+set<int32_t>
+Alphabet::symbolsWhereLeftIs(UChar l) const {
```

Comment:
Feels like that should take a whole code point?

Reply:
currently working on that set of issues - will push shortly